pmoves/docs/operations/YT_EGRESS_RUNBOOK.md

```
@@ -0,0 +1,214 @@
+differently.
+
+## Preflight
+
+Before activation, confirm KVM4-1 is advertising as exit node:
+
+```bash
+make -C pmoves yt-egress-preflight
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)